### ISSUE TITLE
Keyboard navigation on dropdowns does not work?

### ISSUE THREAD

**Issue by christianalfoni** [react-bootstrap/react-bootstrap#361]
Hi,

I have been trying to get the keyboard navigation on the dropdowns to work, without any luck. Using Chrome both locally and on the react-bootstrap site.

Is it supposed to work?

Thanks for making this awesome port!

**Comment by mtscout6** [comment]
I think it should work. I don't have time to tackle it today; but we do take pull requests.

**Comment by christianalfoni** [comment]
I will keep this in mind. If I get a chance to implement it, I will create a pull request :-)
